x/staking/side_handler.go

```diff
@@ -256,7 +256,7 @@ func SideHandleMsgSignerUpdate(ctx sdk.Context, msg types.MsgSignerUpdate, k kee
 	// check signer pubkey in message corresponds
 	expectedPubKey, err := helper.CompressPubKey(eventLog.SignerPubkey)
 
-	if !bytes.Equal(newPubKey.Bytes(), expectedPubKey) {
+	if err != nil || !bytes.Equal(newPubKey.Bytes(), expectedPubKey) {
```

Comment:
should we separate the nil check here because the log below could be misleading?

Reply:
okay

Reply:
fixing this one on staking test cases